.travis.yml

```diff
@@ -50,6 +50,14 @@ jobs:
       php: 7.1
       env: COMPOSER_FLAGS=""
       script: php -n -d memory_limit=768M vendor/bin/phpcs
+    - stage: check coding style
```

Comment:
Do both phpcs and php-cs-fixer need to be used?

Reply:
Not really. In an other project I had situations where phpcs complaint and cs-fixer did not, so I use both. With the cs-fixer I can check more things, but I did not really search for the rules to do the same things with phpcs.

Which one do you prefer?

Reply:
If you think that php-cs-fixer can catch the important coding style issues that you're using phpcs to detect, then I'd say move to php-cs-fixer since it's nice that it can correct a lot of issues automatically.

I don't know that it can catch everything that phpcs checks for though, probably depends on the rule set that browscap is using (in my own experience, we use phpcs in the build to report on code style violations (but don't fail the build on them, since a lot of them are hard to fix across the board, like line-length), and we also use it to lint before creating a PR/Diff.  But we also use php-cs-fixer to automatically fix anything it catches (but don't run this in the build)).

Reply:
codesniffer removed